### DISCUSSION TITLE
Select dycore for next NorESM release

### DISCUSSION THREAD

**Discussion by gold2718** [NorESMhub/CAM#78] [General]
NCAR is focusing on the Spectral Element (SE) dycore for CAM7 physics (prototype for a CMIP7 model). This development includes new vertical schemes and NCAR is not doing any testing or development with the Finite Volume (FV) dycore used until now by NorESM. Before deciding on a final configuration for the next version of NorESM/CAM, we need to decide if we join the SE bandwagon or continue with FV. Here are a few initial thoughts.

**FV**
- Advantages
    - Stable and well-understood dycore
    - Runs fairly fast (less expensive)
- Disadvantages
    - Not currently developed or tested with new vertical schemes
    - Stability issues with higher model tops
    - Limited strong scaling capability

**SE**
- Advantages
    - Good strong and weak scaling capability
    - Currently being developed tested and improved including new energy conservation improvements
    - Development and testing includes higher-top modeling
    - Capability for regionally-refined simulations over selected areas.
    - Capability for faster runs by running physical parameterizations on a lower resolution.
- Disadvantages
    - Slower (~18% more wallclock) and more expensive (~18% more PE-hours / simulated day)

**Comment by MichaelSchulzMETNO** [comment]
Some further points to consider: 
- Input data availability 
- vertical resolution 
- support for the different configurations from NCAR
- output processing
- assimilation capability

**Comment by ingvisau** [comment]
Here is a table comparing cost/time use of different FV and SE runs: [table](https://docs.google.com/document/d/1FG3fU-wsMd5Nv8Ses2creTa_Rd6xqoDlwS6y49jezUI/edit) . Please let me know if there are any questions, or access problems.

**Comment by oyvindseland** [comment]
SE chosen
